static/benchmarks/css/leaderboard/leaderboard-table.sass

```diff
@@ -0,0 +1,190 @@
+.leaderboard-table-component
+  banner
+    display: flex
+    position: relative
+    align-items: center
+    margin: -5px 0 10px 0px
+    margin-bottom: 20px
+
+    .leaderboard-table-banner-icon
+      width: 83px
+      height: 78px
+      z-index: 1
+      border: 5px solid white
+      border-radius: 81px
+
+      img
+        margin: 12px
+
+    .leaderboard-table-banner
+      display: flex
+      align-items: center
+      height: 55px
+      width: 100%
+      margin-left: -15px
+
+      h3
+        margin-left: 10px
+        font-family: 'Open Sans', sans-serif
+        font-size: 28px
+        font-weight: 900
+        color: white
+
+.filters
+    display: flex
+    position: relative
+    height: 80px
+    align-items: center
+    padding-left: 10px
+    margin-bottom: 5px
+
+    .benchmark-sort
+      max-width: 300px
+
+    .legend
+      display: flex
+      position: relative
+      width: 40%
+      min-width: 300px
+      height: 30px
+      align-items: center
+      padding: 0 15px
+      margin-left: auto
+      margin-right: 10px
+      border-radius: 25px
+      background: linear-gradient(90deg, rgba(253.07360206185567, 12.603708989690725, 0.7063979381443353, 0.19576321702511779), rgba(115.52127251251848, 198.4398288659794, 67.740171, 1.0000000000000002))
+
+      label
+        font-weight: bold
+        position: absolute
+        top: -25px
+
+      .legend-right
+        margin-left: auto
+
+    .leaderboard-breadcrumb
+      position: absolute
+      bottom: 0
+      right: 10px
+      font-weight: bold
+
+      .breadcrumb-link
+        &:hover
+          color: #008CBA
+          text-decoration: underline
+
+.leaderboard-table-container
+  overflow-x: scroll
+
+  table
+    table-layout: fixed
+    border-collapse: separate
+    border-spacing: 0 10px
+    padding: 0 10px
+
+    tr
+      height: 80px
+      box-shadow: 0 0.5em 1em -0.125em rgba(10, 10, 10, 0.1), 0 0px 0 1px rgba(10, 10, 10, 0.02)
+      border-radius: 10px
+
+      td
+        vertical-align: middle
```

Comment:
Not sure if this is exactly controlling this, but I think the score itself should be centered in the bubble if possible